### DISCUSSION TITLE
OpenMCT Support

### DISCUSSION THREAD

**Discussion by satzguyz** [nasa/openmct#5126] [Community Q&A] [answered]
Hello

I assume, to control any satellite, one needs to have four integral parts 

1. Ground Terminal
2. Command and control
3. Mission planning (includes sequencing and scheduling )
4. Telemetry processing

Pls, specify what are all areas OpenMct covers.

**Comment by mudinthewater** [comment, marked as answer]
In short: Stock OpenMCT isn't really one of those integral parts of satellite control. That doesn't mean you wouldn't want to include it in a design though. 

Many satellites operated in the past without a web telemetry client. Some would query data from their telemetry database and use tools like Matlab to make plots. Others have full desktop clients that make plots in realtime or recorded venues. 

OpenMCT enhances those tools by providing operators a path to plotting spacecraft data in realtime on the web for analysis. While you  _can_ operate a satellite without it just fine, it enhances your operations capabilities by making plotting more accessible and quicker to the ops team, allowing quicker turnaround for typical operations processes. 

It's worth noting that with OpenMCT's plugin architecture, there's nothing preventing a particular project from expanding on its capabilities and adding one of those integral parts you mentioned - but those would be project adaptations. OpenMCT can be as basic as a recorded telemetry viewer, or as complex as your project wants it to be!
